bait-files/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -34,16 +32,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
 
     UserService userService = UserServiceFactory.getUserService();
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
-    Entity newUser = new Entity("User", request.getParameter("username"));
-    newUser.setProperty("id", userService.getCurrentUser().getUserId());
-    newUser.setProperty("username", request.getParameter("username"));
-    newUser.setProperty("first-name", request.getParameter("first-name"));
-    newUser.setProperty("last-name", request.getParameter("last-name"));
```

Comment:
Store these strings.
ex. private static final String USER = "User";

:D

Reply:
Nvm see that you already did it!

Reply:
:)